tests/ops/qutrit/test_qutrit_matrix_ops.py

```diff
@@ -616,6 +617,24 @@ def test_adjoint(self):
         adjoint_mat = op.data[0].T.conj()
         assert qml.math.allclose(adjoint_op.data[0], adjoint_mat)
 
+    def test_adjoint(self):
+        """Tests the metadata and unitary for an adjoint ControlledQutritUnitary operation."""
+        U1 = unitary_group.rvs(3, random_state=10)
+
+        op = qml.ControlledQutritUnitary(
+            U1, control_wires=("b", "c"), wires="a", control_values="01"
+        )
+
+        adjoint_op = op.adjoint()
+        assert isinstance(adjoint_op, qml.ControlledQutritUnitary)
+
+        assert adjoint_op.hyperparameters["u_wires"] == op.hyperparameters["u_wires"]
+        assert adjoint_op.control_wires == op.control_wires
+        # assert adjoint_op.hyperparameters["control_values"] == op.hyperparameters["control_values"]
```

Comment:
Why is this particular test being added in the `TRZ` PR?

Reply:
It was just something that I had forgotten to update when `ControlledQutritUnitary` was merged. It was a minor enough change that I felt like it was fine to put it in this PR. Admittedly, a big reason for that was also to avoid opening unnecessary PR's as at the time it would take a while for the qutrit PRs to get merged.

Reply:
Okay, fair enough!